### ISSUE TITLE
export to C: input/output functions not replaced

### ISSUE THREAD

**Issue by GitMensch** [fesch/Structorizer.Desktop#403]
Given [monthly.nsd.txt](https://github.com/fesch/Structorizer.Desktop/files/1003506/monthly.nsd.txt)
![grafik](https://cloud.githubusercontent.com/assets/6699539/26094141/024057f2-3a19-11e7-9c41-eb6f7329cccf.png)

and the export settings
![grafik](https://cloud.githubusercontent.com/assets/6699539/26094202/47cb275c-3a19-11e7-8a9b-5a40df15364c.png)

the export is 

```c
// program monthly 
// Generated by Structorizer 3.26-06 

#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main(void)
{
	// TODO: Check and accomplish variable declarations: 
	int month;
	
	// TODO: 
	// For any input using the 'scanf' function you need to fill the first argument. 
	// http://en.wikipedia.org/wiki/Scanf#Format_string_specifications 
	
	// TODO: 
	// For any output using the 'printf' function you need to fill the first argument: 
	// http://en.wikipedia.org/wiki/Printf#printf_format_placeholders 
	
	month = 0;
	input(month);
	switch (month) {
	case 1:
	case 2:
	case 3:
	case 4:
		output("beginn");
		break;
	case 11:
	case 12:
		output("end");
		break;
	default:
		output("between");
	}
	
	return 0;
}
```

Shouldn't the input/output functions be replaced by the printf/sscanf functions as the comments indicate?

**Comment by codemanyak** [comment]
The reason is that the parentheses weren't expexted. Instead, the parser assumes a whitespace between the INPUT/OUTPUT keyword on the one hand and the variable name or expression list, respectively, on the other hand.
Or to say it more clearly: The detection mechanisms for highlighting, executor and generator are not always consitstent. I try to mend it as soon as possible. For the time being just insert a space character after the keywords...

**Comment by codemanyak** [comment]
On that occasion: The export of empty input instructions (#51) to C# also needed fixing.

**Comment by codemanyak** [comment]
Fixed with 3059e6f
